web/themes/custom/karaoke_2022/karaokedesign.theme

```diff
@@ -7,9 +7,14 @@
 
 function karaokedesign_preprocess_bef_radios(array &$variables) {
 
+  $currently_checked = $_GET['sort_by'] ?? '';
```

Comment:
Technically $_GET is against coding standards.

Reply:
Looking through things... $_GET isn't there for ajax calls... In fact, the way BEF works may be putting a kink in the way this theme works.
![image](https://user-images.githubusercontent.com/526491/198044688-e22ee956-ed51-412e-b16b-478b0ae4325b.png)